llvm/lib/Target/AMDGPU/VOP2Instructions.td

```diff
@@ -2520,16 +2520,22 @@ multiclass VOP2_Real_DOT_ACC_gfx10<bits<6> op> :
   VOP2_Real_dpp_gfx10<op>,
   VOP2_Real_dpp8_gfx10<op>;
 
+multiclass VOP2Only_Real_DOT_ACC_gfx10<bits<6> op> : VOP2_Real_dpp_gfx10<op>,
+                                                     VOP2_Real_dpp8_gfx10<op> {
+  let IsSingle = 1 in
```

Comment:
Not sure what IsSingle means

Reply:
https://reviews.llvm.org/D99408 first introduced it. (It is also where I got to know what `_e32` and `_e64` suffices are. :-)